guides/plugins/plugins/framework/data-handling/add-complex-data-to-existing-entities.md

```diff
@@ -230,8 +230,7 @@ CREATE TABLE IF NOT EXISTS `swag_example_extension` (
     `created_at` DATETIME(3) NOT NULL,
     `updated_at` DATETIME(3) NULL,
     PRIMARY KEY (`id`),
-    KEY `fk.swag_example_extension.product_id` (`product_id`),
```

Comment:
This would technically allow to many associations, but when we want to have one-to-one we need to keep this constraints
```
KEY `unique.swag_example_extension.product` (`product_id`, `product_version_id`),
````

Reply:
Wouldnt the statement have to be this to get the desired result then?
```sql
CONSTRAINT `unique.swag_example_extension.product` UNIQUE (`product_id`, `product_version_id`),
```

Since as far as im aware `KEY` on its own is just an alias for `INDEX` which only indexes the given columns and does not make sure that the combination of the 2 values is unique.